SPDY/SPDYStreamManager.m

```diff
@@ -21,8 +21,8 @@ @interface SPDYStreamNode : NSObject
 @implementation SPDYStreamNode
 {
   @public
-    __strong SPDYStreamNode *next;
-    __strong SPDYStreamNode *prev;
+    __weak SPDYStreamNode *next;
+    __weak SPDYStreamNode *prev;
```

Comment:
I guess it looks like you do maintain a separate dictionary (actually 2 dictionaries) for holding onto the nodes.  That's fine, but you've coupled your Node's to requiring that and just holding onto head is not enough to keep a chain alive...I still think you could just make prev being weak.